dist/rpm/openQA.spec

```
@@ -314,6 +314,13 @@ Use this package to install munin scripts that allow to monitor some openQA
 statistics.
 %endif
 
+%package openqa-cli-bash-completion.bash
+Summary:        Bash Completion for %{name}
+Requires:       bash-completion
+Supplements:    (%{name} and bash)
+
+%description openqa-cli-bash-completion
```

and this also needs to be in-sync with the package definition above, i.e.
```suggestion
%description client-bash-completion
```

Note: Although the completion is for openqa-cli, I would go for the package name openQA-client-bash-completion as the openqa-cli is delivered by openQA-client package.